internal/node/node.bzl

```diff
@@ -301,10 +301,8 @@ nodejs_binary = rule(
     executable = True,
     outputs = _NODEJS_EXECUTABLE_OUTPUTS,
 )
-"""
-Runs some JavaScript code in NodeJS.
-"""
 
+# Runs some JavaScript code in NodeJS.
```

Comment:
this will lose the documentation. run `yarn skydoc` to see the corresponding changes in the docs/ folder.

I think we need to enable only a whitelist of buildifier lint rules.

Reply:
working on a fix